chapters/ch29.md

```diff
@@ -1,31 +1,32 @@
 # 29. Plotting training error
 ->
-
+# 29. Vẽ đồ thị sai số huấn luyện
```

Comment:
Tạm thời em cứ dịch "error" là "lỗi" cho thống nhất với toàn bộ các chương trước. Nếu cần đổi anh sẽ cập nhật ở #251